pulser-core/pulser/noise_model.py

```diff
@@ -240,8 +255,8 @@ def to_tuple(obj: tuple) -> tuple:
         )
 
         # Get rid of unnecessary None's
-        for p_ in _POSITIVE | _PROBABILITY_LIKE:
-            param_vals[p_] = param_vals[p_] or 0.0
+        for p_ in param_vals.keys():
+            param_vals[p_] = param_vals[p_] or self._get_default_value(p_)
```

Comment:
You should rather do this to avoid overwriting non-`None` values

```suggestion
        for p_ in param_vals.keys():
            param_vals[p_] = param_vals[p_] or self._get_default_value(p_)
```
```suggestion
        for p_, val in param_vals.items():
            param_vals[p_] = self._get_default_value(p_) if val is None else val 
```